app/controllers/visualizations_controller.rb

```diff
@@ -50,24 +52,33 @@ def visualization
     end
   end
 
+  # TODO: (gdingle): overwrite on save
   def save
     @type = params[:type]
```

Comment:
I just realized this controller should probably use strong params (like visualizations_params with permitted whitelist) instead of using params directly

Reply:
Yea, good point. I only learned about strong param after I wrote this controller.